instrumentation/restlet/restlet-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/restlet/v1_0/RestletSingletons.java

```diff
@@ -16,9 +18,22 @@
   private static final Instrumenter<Request, Response> INSTRUMENTER =
       RestletTracing.create(GlobalOpenTelemetry.get()).getServerInstrumenter();
 
+  private static final ServerSpanNameSupplier<String> SERVER_SPAN_NAME =
+      (context, pattern) -> {
+        if (pattern == null || pattern.equals("")) {
+          return null;
+        }
+
```

Comment:
is this needed? ServletContextPath.prepend handles this case (and still prepends servlet context root)
```suggestion
```

Reply:
Good point -- I'll simplify that.